MAUI/Scheduler/accessibility.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: post
+title: Accessibility in .NET MAUI SfScheduler control | Syncfusion
+description: Learn here about Accessibility support with the Syncfusion .NET MAUI Scheduler (SfScheduler) control, its elements and more.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Accessibility Support in .NET MAUI Scheduler (SfScheduler)
+
+## Keyboard navigation
+
+The `SfScheduler` supports selecting the scheduler date time, appointments, and view navigation using keyboard interaction in WinUI, MAC Catalyst, and iOS platforms.
+
+### Day, Week and WorkWeek views
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Right arrow<br/><br/></td><td>Moves selection to the same time slot on the next (LTR) or previous (RTL) day.<br/><br/></td></tr>
+<tr>
+<td>
+Left arrow<br/><br/></td><td>
+Moves selection to the same time slot on the previous (LTR) or next (RTL) day.<br/><br/></td></tr>
+<tr>
+<td>
+Down arrow<br/><br/></td><td>
+Moves selection to the next time slot directly below the currently selected time slot.<br/><br/></td></tr>
+<tr>
+<td>
+Up arrow<br/><br/></td><td>
+Moves selection to the previous time slot directly above the currently selected time slot.<br/><br/></td></tr>
+<tr>
+<td>
+Enter<br/><br/></td><td>
+It triggers Tapped event<br/><br/></td></tr>
+</table>
+
+### Timeline views
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Right arrow<br/><br/></td><td>Moves selection to the next (LTR) or previous (RTL) time slot of the currently selected time slot. <br/><br/></td></tr>
```

Comment:
Can we mention selection change triggers for each navigation from the arrow key?

Reply:
Added selection trigger for cell selection with API link.